src/packages/fact_manager/fact_manager.py

```diff
@@ -306,7 +306,7 @@ async def add_transaction(self, fact_name: str, fact_lang: str, author: str, msg
                             f"VALUES (%s, %s, %s, %s, %s, %s, %s)")
 
         query_data = (fact_name, fact_lang, author, msg, old_field, new_field,
-                      datetime.datetime.utcnow())
+                      pendulum.now())
```

Comment:
[style] We should be consistent, is it pendulum.now() or pendulum.now(pendulum.tz.UTC).